packages/devtools-launchpad/config/development.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": "Launchpad",
+  "environment": "development",
+  "baseWorkerURL": "http://localhost:8000/assets/build/",
+  "favicon": "assets/launchpad-favicon.png",
+  "theme": "light",
+  "dir": "ltr",
+  "logging": {
+    "client": false,
+    "firefoxProxy": false,
+    "actions": false
+  },
+  "development": {
+    "serverPort": 8000,
```

Comment:
Do we want to use a different port, so as not to conflict with debugger running locally?